CONTRIBUTING.md

```diff
@@ -111,6 +111,52 @@ pnpm run test -m "$STRING_MATCH"
 node --test ./test/foo.test.js
 ```
 
+#### Running a single test
+
+Sometimes you want to run a single test case (`it` or `describe`) or a single test file. You can do so by using Node.js. 
+
+To run a single test file, for example `test/astro-basic.test.js`:
+
+```shell
+node --test test/astro-basic.test.js
```

Comment:
I've always been running without --test, I'm curious to know what difference it makes

Reply:
I don't think there's a difference. However, it's possible that the flag does some optimisations underhood when combined with other options. 

Also, considering that's the [recommended way](https://nodejs.org/api/test.html#running-tests-from-the-command-line), we should suggest it too.